packages/transaction-controller/CHANGELOG.md

```diff
@@ -6,6 +6,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.0.0]
+### Added
+- Add optional `getLastBlockVariations` method to `RemoteTransactionSource` type ([#1668](https://github.com/MetaMask/core/pull/1668))
+- Add `updateTransactionGasFees` public method to `TransactionController` ([#1674](https://github.com/MetaMask/core/pull/1674))
+- Add `r`, `s` and `v`  properties to `TransactionMeta` ([#1664](https://github.com/MetaMask/core/pull/1664))
+- Add `sendFlowHistory` property to transaction meta. ([#1665](https://github.com/MetaMask/core/pull/1665))
```

Comment:
```suggestion
- Add `sendFlowHistory` property the transaction metadata ([#1665](https://github.com/MetaMask/core/pull/1665))
```

Reply:
should this be like this perhaps?

```suggestion
- Add `sendFlowHistory` property to the transaction metadata. ([#1665](https://github.com/MetaMask/core/pull/1665))
```

Reply:
Forgot the `to` but I think we're avoiding `.` in this changelog at least.